frontend/docsettings.lua

```diff
@@ -176,8 +210,17 @@ function DocSettings:open(doc_path)
     end
     local history_file = new:getHistoryPath(doc_path)
 
+    if G_reader_settings:isTrue("document_metadata_hash_enabled") then
+        new.hash_sidecar_dir, new.hash_sidecar_file =
+                new:getSidecarHashDirAndFilepath(doc_path)
+    end
+
     -- Candidates list, in order of priority:
     local candidates_list = {
+        -- Hash or PDF fingerprint-based sidecar file lookup
+        new.hash_sidecar_file or "",
+        -- Backup file of hash or PDF fingerprint-based sidecar file lookup
+        new.hash_sidecar_file and (new.hash_sidecar_file..".old") or "",
```

Comment:
I don't rememeber how this "priority" works, and why doc_sidecar stuff is before dir_sidecar stuff - and/or if it still matters ? @hius07 @NiLuJe ?.
Should these new.hash_sidecar stuff be first?

Reply:
I put this first for testing but have no problem moving it down.

Reply:
Oh, yeah, definitely not first.